aria/parser/presentation/field_validators.py

```diff
@@ -14,12 +14,29 @@
 # limitations under the License.
 
 
+from ...utils.formatting import safe_repr
 from ..validation import Issue
 from .utils import (parse_types_dict_names, report_issue_for_unknown_type,
                     report_issue_for_parent_is_self, report_issue_for_unknown_parent_type,
                     report_issue_for_circular_type_hierarchy)
 
 
+def not_negative_validator(field, presentation, context):
```

Comment:
Python will do the right thing here -- if there is a `__lt__` it will call it, if not it will try to find a `__gt__` and `__eq__` and do the right stuff.